src/modules/addlist.js

```diff
@@ -1,24 +1,62 @@
-import tasksToDo from './Array.js';
-
-export default class ContentList {
-  static displayToDoList = () => {
-    const toDoListContainer = document.querySelector(
-      '.to-do-list',
-    );
-    // sort the tasksToDo array by index
-    tasksToDo.sort((a, b) => a.index - b.index);
-    tasksToDo.forEach((task) => {
-      const listItem = document.createElement('div');
-      listItem.className = 'to-do-list-item';
-      listItem.innerHTML = `<li>
-   <input type="checkbox" class ="to-do-list" id="${task.index}" name="${task.description}" value="${task.description}"> ${task.description}
-  </li>
-  `;
-      const optionBtn = document.createElement('button');
-      optionBtn.className = 'option-btn';
-      optionBtn.innerHTML = '<i class="fa fa-ellipsis-v" aria-hidden="true"></i>';
-      listItem.appendChild(optionBtn);
-      toDoListContainer.appendChild(listItem);
-    });
-  };
+import clearCheckedItems from './delete.js';
+
+const listItems = document.getElementById('list-container');
+
+const InputBox = document.getElementById('input-box');
+
+const clearBtn = document.getElementById('close-btn');
+
+const addTask = document.getElementById('addTask');
+
+const saveData = () => {
+  localStorage.setItem('data', listItems.innerHTML);
+};
+
+function addTask1() {
+  if (InputBox.value === '') {
+    alert('You must write something');
+  } else {
+    const li = document.createElement('li');
+    li.draggable = true;
+    li.innerHTML = InputBox.value;
+    listItems.appendChild(li);
+
+    const span = document.createElement('span');
+    span.innerHTML = '\'<img src="images/trash-can.png" alt="trash">\'';
+    span.innerHTML = '\u00d7';
+    li.appendChild(span);
+  }
+
```

Comment:
- [ ] This is what is saved in your local storage when I add a task: HTML elements.

  ![image](https://github.com/AleWaweru/TODO-LIST/assets/54848363/280c96df-94a2-4e6d-9c39-7f08ec1249f8)

  Kindly note that the local storage should save an array of tasks, each having `description`, `index`, and `completed` as required in the previous project.

- [ ] There is no way to edit the task descriptions. Kindly add the edit feature and ensure that the changes are persisted in the local storage.

Reply:
done